src/ray/util/container_util.h

```diff
@@ -120,6 +139,16 @@ std::ostream &operator<<(std::ostream &os,
                          DebugStringWrapper<absl::flat_hash_map<Ts...>> c) {
   return c.StringifyContainer(os);
 }
+template <typename... Ts>
+std::ostream &operator<<(std::ostream &os,
+                         DebugStringWrapper<std::unordered_map<Ts...>> c) {
+  return c.StringifyContainer(os);
+}
+
+template <typename T>
+std::ostream &operator<<(std::ostream &os, DebugStringWrapper<std::optional<T>> c) {
+  return os << debug_string(c.obj_.value());
```

Comment:
I think you need to check has_value here

Reply:
Fixed.